src/base/preferences.cpp

```diff
@@ -53,9 +55,9 @@
 
 #include "utils/fs.h"
 #include "utils/misc.h"
```

Comment:
> Once you have decided to sort it, then do it right.

You come across as extremely rude. Do you really think I intentionally messed up the order?
 People make mistakes; you need to act in a way that _encourages_ open source contributions, not in such a way that deters them.